azure/src/test/java/org/apache/iceberg/azure/adlsv2/ADLSLocationTest.java

```diff
@@ -38,11 +38,26 @@ public void testLocationParsing(String scheme) {
     assertThat(location.path()).isEqualTo("path/to/file");
   }
 
```

Comment:
Do we have tests for the validation failures?

Reply:
Added a new one [here](https://github.com/apache/iceberg/pull/11294/commits/7f4ad691cbaf946ee7511b6f2dc33f6e885384a9#diff-14a2995e524eeccee0784e3271aae9ba49fde6e45535c5d351a84d44827dfb3eR82-R89)